buildSrc/src/main/java/org/springframework/boot/build/ConventionsPlugin.java

```diff
@@ -111,6 +103,7 @@
  *
  * @author Andy Wilkinson
  * @author Christoph Dreis
+ * @author Mike Smithson
```

Comment:
The javadoc above should be updated to replace the list of actions when the `MavenPublishPlugin` is applied and to link to the new Maven conventions class instead.